package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
what is this?
@Mahimahto looks like two files package.json and package-lock.json have been added here unintentionally.